cypress/support/steps/main.steps.ts

```diff
@@ -49,13 +51,20 @@ export const supply = (
 
   return describe(`Supply process for ${_shortName}`, () => {
     skipSetup({ skip, updateSkipStatus });
-    it(`Open ${_shortName} supply popup view`, () => {
-      doSwitchToDashboardSupplyView();
-      cy.get(`[data-cy='dashboardSupplyListItem_${_shortName.toUpperCase()}']`)
-        .find('button:contains("Supply")')
-        .click();
-      cy.get(`[data-cy=Modal] h2:contains("Supply ${_shortName}")`).should('be.visible');
-    });
+    if (!isDetailsPage) {
+      it(`Open ${_shortName} supply popup view`, () => {
+        doSwitchToDashboardSupplyView();
+        cy.get(`[data-cy='dashboardSupplyListItem_${_shortName.toUpperCase()}']`)
+          .find('button:contains("Supply")')
+          .click();
+        cy.get(`[data-cy=Modal] h2:contains("Supply ${_shortName}")`).should('be.visible');
+      });
+    } else {
+      it('Open via details page', () => {
+        cy.contains('Your info').click();
+        cy.contains('Supply').click();
+      });
+    }
```

Comment:
i guess that u don't need to change this one, cause u never use this functionality